torch/csrc/distributed/c10d/symm_mem/nvshmem_extension.cu

```diff
@@ -416,33 +434,32 @@ void _all_to_all_get_inner(
   auto input_ptr = input.const_data_ptr();
   auto output_ptr = out.mutable_data_ptr();
 
-  // Required args
-  std::vector<void*> args = {
-      &input_ptr,
-      &output_ptr,
-      &src_offsets_ptr,
-      &out_splits_ptr,
-      &dst_offsets_ptr,
-      &stride_bytes,
-      &team_to_use};
-
   // Optional args for signals
-  void *b_start_ptr, *b_len_ptr, *b_head_ptr;
+  void *b_start_ptr = nullptr, *b_len_ptr = nullptr, *b_head_ptr = nullptr;
   if (has_signal) {
```

Comment:
Should at least have a proper assert error message check that has_signal implies all the optional args are not null. Ideally a ValueError error message.